Makefile

```diff
@@ -391,8 +391,8 @@ test_functional: venv
 test_client_lib:
 	@cd client/python/ovmsclient/lib && \
 		make style || exit 1 && \
-		. .venv/bin/activate; make build || exit 1 && \
-		make test || exit 1 && \
+		. .venv-ovmsclient/bin/activate; make build || exit 1 && \
```

Comment:
test_client_lib has no target dependency, how can we ensure that .venv-ovmsclient will be present?

Reply:
.venv_ovmsclient is created in Makefile in ovmsclient dir, so make style creates and configures such venv if it doesn't exist.